uint8 waypoint_id   # sequence number of target waypoint
---
uint8 status        # result code
